### ISSUE TITLE
DDP is up to date. (undef)

### ISSUE THREAD

**Issue by KES777** [garu/Data-Printer#163]
DDP is not updated via `carton`

How to reproduce:

add into `cpanfile` : requires 'DDP'
$ carton update

DDP should have a `$VERSION`

**Comment by garu** [comment]
But DDP does have a `$VERSION`: https://github.com/garu/Data-Printer/blob/main/lib/DDP.pm#L8

```
▶ perl -MDDP -E 'say $DDP::VERSION'
1.000004
```

@skaji any idea what might be going on here?

**Comment by garu** [comment]
@KES777 while we work on the issue, a simple workaround is to use the name of the distribution:

`requires 'Data::Printer'`

**Comment by skaji** [comment]
PAUSE does not recognize such *dynamic* version declaration.

```
❯ curl -s https://cpan.metacpan.org/modules/02packages.details.txt.gz | zgrep '^DDP '
DDP                               undef  G/GA/GARU/Data-Printer-1.000004.tar.gz
```

If you wish PAUSE to extract versions properly, use `our $VERSION = '1.000004';` syntax in DDP.pm too.

Or, you can set "provides" explicitly in META.json.
See, for example, https://metacpan.org/release/SKAJI/App-cpm-0.997004/source/META.json#L74

**Comment by garu** [comment]
Fixed in 1.1.0 (released in 2022-12-22)
